tests/status.py

```diff
@@ -72,6 +72,35 @@ def runTest(self):
         board.expect(prompt)
         self.result_message = 'ifconfig shows ip addresses: %s' % tmp
 
+class MacAddressCheck(rootfs_boot.RootFSBootTest):
+    '''Checked MAC addresses for all interfaces.'''
+    def runTest(self):
+        board.sendline('\nhexdump -n 6 -v -e \'/1 "%02X"\' /proc/device-tree/uccp@18480000/mac-address0')
+        board.expect('0019F5890AFB.*')
+        board.expect(prompt)
+        board.sendline('\nhexdump -n 6 -v -e \'/1 "%02X"\' /proc/device-tree/uccp@18480000/mac-address1')
+        board.expect('0019F5890AFC.*')
+        board.expect(prompt)
+        board.sendline('\nhcitool dev | grep 00:19:F5:89:0A:FD')
```

Comment:
Which is why I had the exact MAC addresses to be checked in first place.  Anyway, moving on, what do you propose now?